segway/output.py

```diff
@@ -12,11 +12,15 @@
 
 from .bed import parse_bed4
 from .layer import layer, make_layer_filename
-from ._util import Copier, maybe_gzip_open
+from ._util import (Copier, maybe_gzip_open, extract_superlabel, 
+                    BED_SCORE, BED_STRAND, NUM_COLORS, SCHEME)
 
 INDEX_BED_START = 1
 INDEX_BED_THICKSTART = INDEX_BED_START + 5
 
+LABEL_INDEX = 3
```

Comment:
there is `BED_SCORE` and `BED_STRAND` constants above. Can they not also be used instead of the `LABEL_INDEX` and/or `INDEX_BED_START`? stuff

Reply:
Can you elaborate on what you mean here? Are you requesting moving these constants to the `_util.py` file and importing them in `output.py`?